dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/sfdr/model/environmental/greenhouseGasEmissions/SfdrEnvironmentalGreenhouseGasEmissions.kt

```diff
@@ -1,56 +1,41 @@
-package org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.sfdr.model.environmental.greenhouseGasEmissions
 
-import jakarta.validation.Valid
 import org.dataland.datalandbackend.model.datapoints.CurrencyDataPoint
 import org.dataland.datalandbackend.model.datapoints.ExtendedDataPoint
 import org.dataland.datalandbackend.model.enums.commons.YesNo
 import java.math.BigDecimal
 
 /**
- * --- API model ---
- * Fields of the subcategory "Greenhouse gas emissions " belonging
- * to the category "Environmental" of the sfdr framework.
-*/
+ * The data-model for the GreenhouseGasEmissions section
+ */
 data class SfdrEnvironmentalGreenhouseGasEmissions(
-    @field:Valid
-    val scope1GhgEmissionsInTonnes: ExtendedDataPoint<BigDecimal>? = null,
+    val scope1GhgEmissionsInTonnes: ExtendedDataPoint<BigDecimal?>? = null,
```

Comment:
Wieso fehlen hier die Valid annotations?

Reply:
Fixed. Valid annotations are now included for all datapoints